src/Logger/Adapter/Raygun.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace Utopia\Logger\Adapter;
+
+use Exception;
+use Utopia\Logger\Adapter;
+use Utopia\Logger\Log;
+use Utopia\Logger\Logger;
+use function curl_error;
```

Comment:
We usually just use `\` in the function call.

Reply:
My bad, PHPStorm likes it this way 😅 Updated.